src/Renci.SshNet/Common/SshData.cs

```diff
@@ -260,8 +267,12 @@ protected string[] ReadNamesList()
         /// <summary>
         /// Reads next extension-pair data type from internal buffer.
         /// </summary>
-        /// <returns>Extensions pair dictionary.</returns>
+        /// <returns>
+        /// Extensions pair dictionary.
+        /// </returns>
+#pragma warning disable CA1859 // Use concrete types when possible for improved performance
         protected IDictionary<string, string> ReadExtensionPair()
+#pragma warning restore CA1859 // Use concrete types when possible for improved performance
```

Comment:
Seems fine to apply this fix?
```suggestion
        protected Dictionary<string, string> ReadExtensionPair()
```